resources/kyverno/admission-control-policy/released/verify-attestation.json

```diff
@@ -1,21 +1,11 @@
 {
-    "apiVersion": "kyverno.io/v1",
-    "kind": "ClusterPolicy",
-    "metadata": {
-        "annotations": {
-            "pod-policies.kyverno.io/autogen-controllers": "none"
-        },
-        "name": "attest-code-review"
-    },
     "spec": {
-        "validationFailureAction": "enforce",
-        "background": false,
-        "webhookTimeoutSeconds": 30,
-        "failurePolicy": "Fail",
         "rules": [
             {
+                "name": "attest-code-review",
                 "verifyImages": [
                     {
+                        "image": "ttl.sh/*",
```

Comment:
any way to tighten this up?

Reply:
This was done as some of the examples use a random name to store the image:
```
DEFAULT_REPOSITORY=$(xxd -l 16 -c 16 -p < /dev/random)
: "${REGISTRY:=ttl.sh}"
: "${REPOSITORY:=$REGISTRY/$DEFAULT_REPOSITORY}"
```

Reply:
We could put a more specific image pattern here if we had a good idea of the use cases we want to support. There was some discussion yesterday about setting up different policies for different namespaces.

Reply:
I agree. This policy needs to be revised in a future PR but should not be a blocker for this. I created an issue to track this needed change #113

Reply:
I do think we should separate out example policies in some way. Whether that's other namespaces or what.

Reply:
Cool - let's track it there - thanks